routes/imageAPI.js

```diff
@@ -0,0 +1,30 @@
+var imageController = require('../controllers/imageController');
+var router = require('express').Router();
+
+//body parameters
+//	required:
+//		url:String
+//		type:String (user,idea,team,project,other)
+//		name:String
+//response:
+//{image:ImageEntity}
+router.post('/add',imageController.addNewImage);
+
+//body parameters
+//	required:
+//		id:String
+//response:
+//{image:ImageEntity}
+router.get('/id/:id',imageController.getImageById);
+
+//query parameters
+//	required:
+//		type:String 
```

Comment:
the comment is wrong. actually "type" is in the path variable, not the query parameters.